app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -19,8 +19,10 @@ def get_timezones(_options = {})
   end
 
   def allowed_sys_type(_options = {})
-    # TODO: replace with api provided values, once issue '114' is solved and merged
-    {0 => "s922", 1 => "e880"}
+    ar_ems = ar_ems_get
+    ar_sys_types = ar_ems&.flavors
```

Comment:
> ar_ems&.flavors

If we can't find the EMS I wonder if we should just raise an error instead of trying to handle it here, that would be catastrophic down the line anyway

Reply:
i did the corresponding changes.